002-storage.md

```diff
@@ -0,0 +1,186 @@
+# Storage fleet
+
+# **Storage fleet**
```

Comment:
Some preamble here describing how PG organizes files here and where chunks fit in would make it easier to read/understand for someone new.

Reply:
Yeah, that's in todo.